app/Traits/BibleFileSetsTrait.php

```diff
@@ -273,7 +274,10 @@ function ($filesetProperty) {
         } else {
             // Multiple files per chapter
             $hasMultiMp3Chapter = $this->hasMultipleMp3Chapters($fileset_chapters);
-            if (sizeof($fileset_chapters) > 1 && !$is_video && $hasMultiMp3Chapter) {
+            if (sizeof($fileset_chapters) > 1 &&
```

Comment:
the logic can be simplified here. Instead of not video and not usx, the check could be for audio.  The checks for multichapter etc are still valid. I'm just speaking about video and usx checks

Reply:
@bradflood this logic has been simplified as you have suggested. You can see it in the next line:

https://github.com/faithcomesbyhearing/dbp/pull/753/files#diff-cd91d7f15ddfe5fa978e89a8db72344c852408d540765d7f0ae9a85e343aae24R274